src/test/storage/encoded_segment_test.cpp

```diff
@@ -171,7 +185,7 @@ TEST_P(EncodedSegmentTest, SequentiallyReadNotNullableIntSegment) {
 
 TEST_P(EncodedSegmentTest, SequentiallyReadNullableIntSegment) {
   auto value_segment = create_int_with_null_value_segment();
-  auto base_encoded_segment = encode_value_segment(DataType::Int, value_segment);
+  auto base_encoded_segment = encode_base_segment(DataType::Int, value_segment);
```

Comment:
Just "encode_segment"?

Reply:
Did that, but for some reason clang does no longer find the eventual `encode_segment` method in the utils, even though the signature differ. Might be related to the other problem where I use `::opossum::`?

Any ideas what I am doing wrong?

Reply:
Hard to check without access to a compiler. Not a.big thing anyway.